### ISSUE TITLE
[On Hold] Allow dates to be inherited for ASpace objects

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1803]
**Story**

Currently, the only way for a date to display in DCS for objects associated with ArchivesSpace, is if a date is entered directly at the corresponding archival object level in ArchivesSpace. In Archives at Yale, when no date is present at the archival object level, it is inherited from a the most immediate higher level of description e.g series or collection, where a date can be found. We would like to replicate this mapping in DCS. 

**Acceptance**
- [x] When not date is present at the item level for an object in ArchivesSpace, date is mapped from the most immediate higher level of description in ArchivesSpace to DCS.  

**Note:**
This needs to be a Metadata Cloud change.  We need to keep the data distinct so we know which was inherited.  Need to discuss how this is modeled.

**Comment by DraxIndustries79** [comment]
This is waiting on an MC change.  Alicia is submitting an issue to the MC repo this week (2/14)

**Comment by martinlovell** [comment]
Metadata cloud test and uat have been updated.
If it looks good, we can deploy to production a week from Wednesday.

I'm not sure which archival objects to test for DCS.

**Comment by motropuk** [comment]
https://collections-uat.library.yale.edu/catalog/900057352

Looks like this is working.
